src/botPage/view/View.js

```diff
@@ -337,7 +337,7 @@ export default class View {
 
         $('#continue-trading').click(() => {
             const time = parseInt($('#realityDuration').val());
-            if (time >= 10 && time <= 120) {
+            if (time >= 10 && time <= 60) {
```

Comment:
These changes don't seem to require the changes from other PRs. Could you please branch from master and add the changes there?